envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -138,18 +144,19 @@ def test_zerologon_exploiter(self, island_client):
             "aad3b435b51404eeaad3b435b51404ee",
             "2864b62ea4496934a5d6e86f50b834a5",
         ]
-        raw_config = IslandConfigParser.get_raw_config(Zerologon, island_client)
+        serialized_config = IslandConfigParser.get_serialized_config(zerologon_test_configuration)
```

Comment:
Same comment as above. Let's just modify everything to use the object. The only time it needs to be serialized is when we send it to the API.